dll/ntdll/dispatch/amd64/dispatch.S

```diff
@@ -195,7 +200,21 @@ PUBLIC KiUserExceptionDispatcher
     /* Clear direction flag */
     cld
 
+    /* Check the WOW64 callback */
+    mov rax, qword ptr Wow64PrepareForException[rip]
+    test rax, rax
+    jz .NoWow64
+
+    /* Prepare for WOW64 exception dispatching */
+    lea rcx, [rsp + CONTEXT_FRAME_LENGTH] /* ExceptionRecord */
+    lea rdx, [rsp] /* ContextRecord */
+    call rax
+
+.NoWow64:
+
     /* Dispatch the exception */
+    lea rcx, [rsp + CONTEXT_FRAME_LENGTH] /* ExceptionRecord */
+    lea rdx, [rsp] /* ContextRecord */
     call RtlDispatchException
 
     /* Check for success */
```

Comment:
Comment for lines 224-227 (the ZwContinue -- I cannot add comment there with the github interface)
Shouldn't this ZwContine call be a `RtlRestoreContext(ContextRecord);` instead, on x64 only?